services/player-service/application/src/main/java/com/kjeldsen/player/application/repositories/PlayerPositionTendencyReadRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.kjeldsen.player.application.repositories;
+
+import com.kjeldsen.player.domain.PlayerPosition;
+import com.kjeldsen.player.domain.PlayerPositionTendency;
+
+import java.util.List;
+
+public interface PlayerPositionTendencyReadRepository {
+    PlayerPositionTendency get(PlayerPosition position);
```

Comment:
let's discuss get here... not sure if it should be Optional< PlayerPositionTendency>?

Reply:
We are never going to have a null, because the implementation of this method is retrieving stored one or default one.